README.md

```diff
@@ -1 +1,49 @@
-# sldt-semantic-models
+# Aspect Models for Eclipse Tractus-X Semantic Layer (SLDT)
+The repository contains the aspect models based on [BAMM](https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/snapshot/index.html) for the Tractus-X project in Catena-X.
+
+
+# Using the models
+The models can locally be processed with the [BAMM CLI](https://openmanufacturingplatform.github.io/sds-documentation/sds-documentation/index.html).
+It allows you to generate different artifacts (diagrams, example payload, java class files) out of it.
+
+To generate the commonly used set of artifacts, this repository also contains a script, `generate.sh`, which automates some steps, and might be useful for your task.
```

Comment:
I haven't add this script as it does not work with the latest version of the CLI, ie the command line arguments have changed and we need to adjust

Reply:
Thanks for the hint, I removed the remark.